services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceControllerMultiClusterConfig.java

```diff
@@ -286,4 +286,8 @@ public long getServiceDiscoveryRegistrationRetryMS() {
   public List<String> getControllerInstanceTagList() {
     return getCommonConfig().getControllerInstanceTagList();
   }
+
+  public boolean getUpdateRealTimeTopic() {
```

Comment:
Can we make it per cluster config to make rollout more granular?

Reply:
it is defined in `[VeniceControllerClusterConfig.java` `https://github.com/linkedin/venice/pull/1404/files#diff-cea201d965375cda81510a0338bc355594295fd736cfff25579b9d9b97218c6a` 
what else to be done to make it per cluster config?

Reply:
For per cluster we can't use `getCommonConfig().getUpdateRealTimeTopic();`; we need to get configs for that cluster first and then lookup  this field

e.g. `VeniceControllerClusterConfig clusterConfig = multiClusterConfigs.getControllerConfig(clusterName);`

Reply:
done